assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -293,25 +297,38 @@
         </resources>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
+                <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>analyze</id>
+                        <id>add-stack</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
                         <configuration>
-                            <skip>true</skip>
+                            <target>
```

Comment:
I should edit original file and add che-in-che to json file. Do you know how to implement it with leaving it in jar file?